sklearn/metrics/regression.py

```diff
@@ -90,7 +90,10 @@ def _check_reg_targets(y_true, y_pred, multioutput):
     n_outputs = y_true.shape[1]
     multioutput_options = (None, 'raw_values', 'uniform_average',
                            'variance_weighted')
-    if multioutput not in multioutput_options:
+    if isinstance(multioutput, string_types) and multioutput not in multioutput_options:
```

Comment:
When any option among `raw_values`, `uniform_average` or 'variance_weighted' is passed, this condition is not met since it checks for invalid `multioutput_option`. We don't have any check for `multioutput` being a string type *and* a valid `multioutput_option`. That check needs to be added.